ports/qt5-base/cmake/qt_fix_makefile_install.cmake

```diff
@@ -1,34 +1,33 @@
 #Could probably be a vcpkg_fix_makefile_install for other ports?
 function(qt_fix_makefile_install BUILD_DIR)
     #Fix the installation location
-    file(TO_NATIVE_PATH "${CURRENT_INSTALLED_DIR}" NATIVE_INSTALLED_DIR)
-    file(TO_NATIVE_PATH "${CURRENT_PACKAGES_DIR}" NATIVE_PACKAGES_DIR)
-    
-    if(WIN32)
-        string(SUBSTRING "${NATIVE_INSTALLED_DIR}" 2 -1 INSTALLED_DIR_WITHOUT_DRIVE)
-        string(SUBSTRING "${NATIVE_PACKAGES_DIR}" 2 -1 PACKAGES_DIR_WITHOUT_DRIVE)
+    set(MSYS_HACK "")
+    if(VCPKG_HOST_IS_WINDOWS)
+        file(TO_NATIVE_PATH "${CURRENT_INSTALLED_DIR}" NATIVE_INSTALLED_DIR)
+        file(TO_NATIVE_PATH "${CURRENT_PACKAGES_DIR}" NATIVE_PACKAGES_DIR)
+        if(VCPKG_TARGET_IS_MINGW)
+            set(MSYS_HACK ":@msyshack@%=%")
+            file(STRINGS "${BUILD_DIR}/Makefile" using_sh REGEX "= rm -f")
+            if(using_sh)
+                set(NATIVE_INSTALLED_DIR "${CURRENT_INSTALLED_DIR}")
+                set(NATIVE_PACKAGES_DIR "${CURRENT_PACKAGES_DIR}")
+            endif()
+        endif()
         string(SUBSTRING "${NATIVE_INSTALLED_DIR}" 0 2 INSTALLED_DRIVE)
         string(SUBSTRING "${NATIVE_PACKAGES_DIR}" 0 2 PACKAGES_DRIVE)
+        string(SUBSTRING "${NATIVE_INSTALLED_DIR}" 2 -1 INSTALLED_DIR_WITHOUT_DRIVE)
+        string(SUBSTRING "${NATIVE_PACKAGES_DIR}" 2 -1 PACKAGES_DIR_WITHOUT_DRIVE)
     else()
-        set(INSTALLED_DRIVE)
-        set(PACKAGES_DRIVE)
-        set(INSTALLED_DIR_WITHOUT_DRIVE ${NATIVE_INSTALLED_DIR})
-        set(PACKAGES_DIR_WITHOUT_DRIVE ${NATIVE_PACKAGES_DIR})
+        set(INSTALLED_DRIVE "") 
+        set(PACKAGES_DRIVE "")
+        set(INSTALLED_DIR_WITHOUT_DRIVE "${NATIVE_INSTALLED_DIR}")
+        set(PACKAGES_DIR_WITHOUT_DRIVE" ${NATIVE_PACKAGES_DIR}")

```

Comment:
There is a typo here - the space after quotes. Maybe it will break the replacement later.

Reply:
Good catch. This is the current problem.